apps/i18n/common/en_us.json

```diff
@@ -1670,7 +1670,7 @@
   "loginInfo_joinStep1": "Create a Code.org account if they haven’t already done so. They can do this at {url}. Note that they can either sign up with an email address and password, or sign up through Google, Facebook, or Microsoft by clicking on one of these buttons:",
   "loginInfo_joinStep1Buttons": "Screenshot of three buttons, reading 'Continue with Google', 'Continue with Facebook', and 'Continue with Microsoft'",
   "loginInfo_joinStep2": "Sign in to their Code.org account.",
-  "loginInfo_joinStep3": "Navigate to {url} and type in their section code: {code}.",
```

Comment:
@mgc1194, do you know whether it’s better to keep unused strings or delete them?

Reply:
We don't want unused strings in the code unless they are going to be used in the future.
This is the right move. The old string is removed, and a new string is added with a new key.

Reply:
Thanks @mgc1194!